src/Microsoft.AspNetCore.Http/HeaderDictionary.cs

```diff
@@ -97,6 +98,34 @@ public HeaderDictionary(int capacity)
             set { this[key] = value; }
         }
 
+        public long? ContentLength
+        {
+            get
+            {
+                long value;
+                var rawValue = this[HeaderNames.ContentLength];
```

Comment:
Ah yes, I should have read your fir comment more carefully before looking at the code.